WordPressShared/Core/Utility/NSDate+Helpers.swift

```diff
@@ -127,17 +145,19 @@ extension Date {
     /// - Example: 2 days ago
     /// - Example: Jan 22, 2017
     ///
-    public func mediumString(timeZone: TimeZone? = nil) -> String {
-        let relativeFormatter = TTTTimeIntervalFormatter()
+    public func toMediumString(inTimeZone timeZone: TimeZone? = nil) -> String {
+        let relativeFormatter = RelativeDateTimeFormatter()
+        relativeFormatter.dateTimeStyle = .named
+
         let absoluteFormatter = DateFormatters.mediumDate
-        
+
         if let timeZone = timeZone {
             absoluteFormatter.timeZone = timeZone
         }
 
         let components = Calendar.current.dateComponents([.day], from: self, to: Date())
         if let days = components.day, abs(days) < 7 {
-            return relativeFormatter.string(forTimeInterval: timeIntervalSinceNow)
+            return relativeFormatter.localizedString(fromTimeInterval: timeIntervalSinceNow)
```

Comment:
Quick note that another thing this might change compared to the old one is this will say `tomorrow` instead of `in 1 day` (same as your "yesterday" vs "1 day ago", but in other direction, since `abs(days)` works both ways). I think that's ok too (probably unlikely for us to handle a lot of dates in the future I guess… except maybe for showing publishing date of posts that we schedule to be posted in the future) since as you say, better use system-provided localizations for those cases anyway for consistency, but just wanted to highlight this case too.

Reply:
Yeah for sure – added it to the tests in dc42ef1a406a462186ae44cd4b97fc2be8fe4424